.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+# Description
+
+<!-- Please describe the changes included in this PR here. -->
+
+# To-Dos
+
+<!-- Before submitting this PR, please lint and build your changes locally. -->
+<!-- Add an 'x' between the brackets to mark each checkbox as checked. -->
+
+- [ ] Lint and test with `tox`
+
```

Comment:
```suggestion
- [ ]  PR includes a test that will fail if the code is reverted, but the test is not.

```

Reply:
How about:

```
- [ ] Test coverage for new code
```

Your suggestion is a little difficult to understand.

Reply:
It is probably related but different.
If it is too complex to understand - let's not add it.